test/tables/gsub.js

```diff
@@ -363,4 +363,120 @@ describe('tables/gsub.js', function() {
             ]
         }), expectedData);
     });
+
+    //// Lookup type 4 ////////////////////////////////////////////////////////
+    // is already tested above
```

Comment:
That one I guess https://github.com/opentypejs/opentype.js/pull/453/files#diff-69ff5caba80b4c351f1e25cb532d195b016fbe21b912a272d1820a2bd99ea8eaR107-R130